### ISSUE TITLE
INFO: PropertyNotSetInConstructor and INFO: MissingConstructor for all entities 

### ISSUE THREAD

**Issue by michnovka** [weirdan/doctrine-psalm-plugin#129]
Is it expected that these are reported as INFO? Or should they not appear in psalm errors at all.

I am running Symfony 6.1.6, psalm 4.30.0 and weirdan/doctrine-psalm-plugin 2.5.0

```

INFO: MissingConstructor - src/Entity/SupportTicketDepartment.php:17:17 - App\Entity\SupportTicketDepartment has an uninitialized property App\Entity\SupportTicketDepartment::$id, but no constructor (see https://psalm.dev/073)
    private int $id;

...

INFO: PropertyNotSetInConstructor - src/Entity/SupportTicketMailBreakLine.php:28:19 - Property App\Entity\SupportTicketMailBreakLine::$admin is not defined in constructor of App\Entity\SupportTicketMailBreakLine or in any private or final methods called in the constructor (see https://psalm.dev/074)
    private Admin $admin;
```

**Comment by Grzyb9k** [comment]
I have the same problem, these settings don't work:
```xml  
    <issueHandlers>
        <PropertyNotSetInConstructor>
            <errorLevel type="suppress">
                <directory name="src/Entity" />
            </errorLevel>
        </PropertyNotSetInConstructor>
    </issueHandlers>
```
